invenio_app_ils/records/resolver/jsonresolver/document_item.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Resolve the EItem referenced in the Document."""
```

Comment:
Docs and comments in this file need to be updated with `Item` instead of `EItem`.